### DISCUSSION TITLE
Few random ideas

### DISCUSSION THREAD

**Discussion by Tsukino-uwu** [tanjeffreyz/auto-maple#84] [Ideas]
- Option to hide/show all points when editing, highlight currently selected

- Drag around points y,x location directly with the edit gui
 (move the circles around visually instead of having to edit values) 
^also able to lock/unlock moving them, to avoid accidents
and able to lock either X or Y when moving so you don't need correct both values after

- more directional command/routine options like:
left/right random direction
left/right for X duration 
left then right
right then left
X skill this rotation, Y skill next rotation loop (could be useful for those long cd 180sec+ skills to not use more than 1 per rotation)
^for example Domain the first time you reach a routine component, and then Oni Legion the next time instead of Domain

- Optional setting to set default command & routine to load when auto maple is started
for example kanna, and that 1 map you will spend 100+ hours at

- Sort routines by class & level range
( at least think this would make sense unless there is a plan to make all routines work for all classes ? )
would make it a bit easier to find the routine you need instead of having to memorize/learn all map abbreviations

- Auto use of legion/potion wealth ( could maybe be added similar to buffs, 10,20,30,120min cd ? )

- automatic loot rotation for routines somehow ? 
maybe loot points that will get prioritized before loot expire if the player haven't passed that area during the normal command points?
or image detection for loot but think that would prob cause lag ?

- Some way for routines to automatically place/replace summons according to duration/cooldown, so it will get placed even if the routine list is long/short
^Basically handle summon/s in a similar way to moving around the map, for less downtime

- Optional loud alert for players instead of only small beep 
( for paranoid people, that don't want to risk being afk if someone decides to chat after entering the map )

- Friend/guild/alliance player alert 
( only saw a red color in the assets so at least assuming that these don't have warnings ?)

**Comment by ScottBouloutian** [comment]
These are some great ideas, I will add one to the list. It would be great if the rune solver could solve for the runes with rotating arrows. These occur very rarely ostensibly to try to make it harder for these kinds of automated rune solvers. I'm willing to help contribute to this project but I'm not sure where to start.

**Comment by noobishh** [comment]
hey wetdragon, I was wondering if you ahve discord? It will be amazing if you can please guide me through some of these. I've seen you post since the beginning of this github. I feel like you have great insight on how to help. Thank you!
